server/controllers/semantic-checker.js

```diff
@@ -172,7 +28,7 @@ var addWarning = function (workbook, message) {
     if (warningsCount < MAX_WARNINGS) {
         addMessageToArray(workbook.warnings, message);
     } else {
-        addMessageToArray(workbook.errors, errorList.warningsCountError);
+        addMessageToArray(workbook.errors, constants.errors.warningsCountError);
         return false;
     }
 };
```

Comment:
This appears to be a fairly large commented-out block. Let’s mark this for full deletion once this refactor has been put through more use.